GooglePlayInstant/Editor/PlayInstantLoadingScreenGenerator.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.IO;
+using UnityEditor;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.UI;
+
+namespace GooglePlayInstant.Editor
+{
+    /// <summary>
+    /// Class that generates Unity loading scenes for Instant Apps.
+    /// </summary>
+    public class PlayInstantLoadingScreenGenerator
+    {
+        /// <summary>
+        /// Creates a scene in the current project that acts as a loading scene until assetbundles are
+        /// downloaded from the CDN. Takes in a loadingScreenImagePath, a path to the image shown in the loading scene,
+        /// and an assetbundle URL.
+        /// </summary>
+        public static void GenerateLoadingScreenScene(string loadingScreenImagePath, string assetBundleUrl)
+        {
+            if (!File.Exists(loadingScreenImagePath))
+            {
+                Debug.LogError("Loading screen image file cannot be found.");
+            }
+            else
+            {
+                // Removes the loading scene if it is present
```

Comment:
// Removes the loading scene if it is present, otherwise does nothing.

Reply:
Resolved.